civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -6700,3 +6718,15 @@ function civihr_employee_portal_load_html_file($template = '', $moduleName) {
     return '';
   }
 }
+
+/**
+ * Implements hook_page_build()
+ */
+function civihr_employee_portal_page_build(&$page) {
```

Comment:
why do we need this? We already have pages which opens ctools modals, and we didn't need this for them

Reply:
This page is built with a Panels Page, other pages where built with custom pages from code.

Reply:
Other pages have a callback to build the pages, now this callback is managed from Panels.

Reply:
Ok, please explain this in the code also, using a comment

Reply:
Sure